### PR TITLE
PayU Latam: Add additional mandatory fields.

### PR DESCRIPTION
**Pull request by deedeelavinder**
Remote:
15 tests, 39 assertions, 2 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
86.6667% passed
(These two failures both have a responseCode of 'INTERNAL_PAYMENT_PROVIDER_ERROR' and pre-date these changes.)

Unit:
17 tests, 71 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: lib/active_merchant/billing/gateways/payu_latam.rb [hunk 1]
```diff
@@ -236,6 +253,9 @@ def add_payer(post, options)
           payer[:fullName] = address[:name]
           payer[:contactPhone] = address[:phone]
           payer[:dniNumber] = options[:dni_number]
+          payer[:dniType] = options[:dni_type]
```
Comment (dtykocki) [inline lib/active_merchant/billing/gateways/payu_latam.rb]: Should this have a guard like we do in https://github.com/deedeelavinder/active_merchant/blob/dbd6f141c6ae0e3db15d3b3f37d0bf50f5c79663/lib/active_merchant/billing/gateways/payu_latam.rb#L163?

Comment (deedeelavinder) [inline lib/active_merchant/billing/gateways/payu_latam.rb]: 👍

File: lib/active_merchant/billing/gateways/payu_latam.rb [hunk 2]
```diff
@@ -174,8 +181,18 @@ def add_invoice(post, money, options)
         tx_value[:value] = amount(money)
         tx_value[:currency] = options[:currency] || currency(money)
 
+        tx_tax = {}
+        tx_tax[:value] = options[:tx_tax] || '0'
```
Comment (bdewater) [inline lib/active_merchant/billing/gateways/payu_latam.rb]: Other gateways seem to use `options[:tax]`

Comment (curiousepic) [inline lib/active_merchant/billing/gateways/payu_latam.rb]: @deedeelavinder True, the `tx` for `tx_tax` and `tx_tax_base` is extraneous from the perspective of the transactor, probably worth changing.

Comment (deedeelavinder) [inline lib/active_merchant/billing/gateways/payu_latam.rb]: 👍

**Pull request closed without merging**